platform/src/common/components/Dropdowns/OrganizationDropdown.jsx

```diff
@@ -12,7 +12,7 @@ import { useGetActiveGroup } from '@/core/hooks/useGetActiveGroupId';
 
 const cleanGroupName = (name) => {
   if (!name) return '';
-  return name.replace(/-/g, ' ').toUpperCase();
+  return name.replace(/[-_]/g, ' ').toUpperCase();
```

Comment:
Hi @Codebmk / @OchiengPaul442 ...in the near future, we might want to consider something which captures more edge cases beyond just "-" and "_".....we might want to have a more "inclusive" regex pattern in the near future PR since this PR is now already merged.

Reply:
Hi @Codebmk / @OchiengPaul442 , something like below:

`return name.replace(/[^a-zA-Z]/g, ' ').toUpperCase();`

**Key points:**
1. `[^a-zA-Z]` matches any character that is not a letter (uppercase or lowercase)
2. The `^ `inside square brackets means "not"
3. This will replace all non-letter characters (including numbers, punctuation, symbols) with a space
4. Follows the same `.toUpperCase()` transformation as the original code